Source/EasyNetQ.Tests/ConsumeTests/When_a_nack_received_from_the_message_handler.cs

```diff
@@ -25,7 +26,7 @@ public void Should_dispose_of_the_consumer_error_strategy_when_the_bus_is_dispos
     {
         MockBuilder.Dispose();
 
-        ConsumerErrorStrategy.Received().Dispose();
+        ((IDisposable)ConsumerErrorStrategy.Received()).Dispose();
```

Comment:
I think that all of these `Should_dispose_of_the_consumer_error_strategy...` test are not required any more. `IConsumerErrorStrategy` is a singleton and will be disposed with the container. Besides it's the only interface tested that way.

We can then remove substituting `IDisposable` to it.

Reply:
@pziezio nice observation. @sungam3r Could you address it in the separate PR?

Reply:
@pziezio  I know. I just wanted to demonstrate core idea without other changes in tests or something else.

Reply:
#1524